tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -985,8 +985,7 @@ public function testListDocuments(array $data): array
             'content-type' => 'application/json',
             'x-appwrite-project' => $this->getProject()['$id'],
         ], $this->getHeaders()), [
-            'orderAttributes' => ['releaseYear'],
-            'orderTypes' => ['ASC'],
+            'queries' => [ 'orderAsc("releaseYear")' ],
```

Comment:
Just a thought, we should ensure all SDKs are using collection types that enforce string ordering, for cases like multiple ordering queries. I think they are... but would be good to double check

Reply:
cc @lohanidamodar

Reply:
I checked this, all are ordered 👍